src/scenes/Main/components/Buttons.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import { inject } from 'mobx-react';
+
+import { Button } from '@game/components';
+import { GameModel } from '@models/game';
+
+interface IButton {
+    gameModel?: GameModel;
+}
+
+export const Buttons: FC<IButton> = inject('gameModel')((props) => {
+    const { resetGame, stepBack, changeTurn } = props.gameModel as GameModel;
+
+    return (
+        <div className="game-buttons">
+            <Button
+                options={{
```

Comment:
i think it was better to separate button's props into separate object and just use it like buttons.resetGame

Reply:
Good idea, am just thinking about that. I will implement it in the further PRs.